### DISCUSSION TITLE
resource not found error !!!

### DISCUSSION THREAD

**Discussion by CallMeDang123** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3577] [Q&A] [answered]
![error](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/137093557/a7ce83e8-20f7-4426-9fb6-2fb4444f2270)

**Comment by nicolaihenriksen** [comment, marked as answer]
@CallMeDang123 What does your `App.xaml` look like? You need to have the base resource dictionaries imported for `{StaticResource MaterialDesignXyz}` to work...

Example from a working `App.xaml`:
```xaml
<Application ...
             xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes">
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <materialDesign:BundledTheme BaseTheme="Inherit"
                                             ColorAdjustment="{materialDesign:ColorAdjustment}"
                                             PrimaryColor="DeepPurple"
                                             SecondaryColor="Lime" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign2.Defaults.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
```

**Reply by CallMeDang123** [reply]
thank you i think i have solved my issue <3
this is my app.xaml in case you wonder
<Application ...
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Light.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign2.Defaults.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Primary/MaterialDesignColor.DeepPurple.xaml" />
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Secondary/MaterialDesignColor.Lime.xaml" />
                <ResourceDictionary Source="/ResourceXAML/MainResource.xaml"></ResourceDictionary>
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
